specs/predict/common/schemas/Params.yml

```
@@ -9,6 +9,14 @@ allParams:
     - $ref: '#/modelsToRetrieveParam'
     - $ref: '#/typesToRetrieveParam'
 
+fetchAllUsersParams:
```

```suggestion
fetchAllUserParams:
```
the operationId is called `fetchAllUserProfiles`

---

[c1cfe7e](https://github.com/algolia/api-clients-automation/pull/974/commits/c1cfe7ee691b3737ac5691a5458761e6fa560536)

---

I have renamed to match the operationId